design-patterns/gc-intro/index.json-ld

```diff
@@ -0,0 +1,224 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Introduction block - Documentation and working example",
+		"fr": "Bloc d’introduction - Documentation et example pratique"
+	},
+	"description": {
+		"en": "Guidance about using the introduction block pattern on Canada.ca.",
+		"fr": "Orientation sur l’utilisation de la configuration de conception du bloc d’introduction sur Canada.ca."
+	},
+	"modified": "2023-08-16",
+	"componentName": "gc-intro",
+	"status": "stable",
+	"version": "1.0",
+	"pages": {
+		"docs": [
+			{
+				"title": "Introduction block",
+				"language": "en",
+				"path": "gc-intro-doc-en.html"
+			},
+		{
+				"title": "Bloc d’introduction",
+				"language": "fr",
+				"path": "gc-intro-doc-fr.html"
+			}
+		],
+		"examples": [
+			{
+				"title": "Introduction block",
+				"language": "en",
+				"path": "gc-intro-en.html"
+			},
+			{
+				"title": "Bloc d’introduction",
+				"language": "fr",
+				"path": "gc-intro-fr.html"
+			},
+			{
+				"title": "Introduction block with image",
+				"language": "en",
+				"path": "gc-intro-image-en.html"
+			},
+			{
+				"title": "Bloc d’introduction avec image",
+				"language": "fr",
+				"path": "gc-intro-image-fr.html"
+			}
+		]
+	},
+	"dependencies": [
+		"bg-cover"
+	],
+	"a11yGuidance": "No accessibility guidance.",
+	"variations": [
+		{
+			"name": {
+				"en": "Introduction block - default",
+				"fr": "Bloc d'introduction - par défaut"
+			},
+			"status": "stable",
+			"description": {
+				"en": "The introduction block pattern introduces the content of a landing page.",
+				"fr": "La configuration de conception du bloc d’introduction introduit le contenu d’une page d’accueil."
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/common-design-patterns/intro-block.html",
+				"fr": "https://conception.canada.ca/configurations-conception-communes/block-introduction.html"
+			},
+			"iteration": "_:iteration_intro_1",
+			"example": [
+				{
+					"en": { "href": "gc-intro-en.html", "text": "Introduction block" },
+					"fr": { "href": "gc-intro-fr.html", "text": "Bloc d'introduction" }
+				}
+			],
+			"implementation": [
+				"_:implement_intro"
+			],
+			"history": [
+				{
+					"en": "March 2024 - Initial implementation of the variation.",
+					"fr": "Mars 2024 - Implémentation initiale de la variante."
+				}
+			]
+		},
+		{
+			"name": {
+				"en": "Introduction block with image",
+				"fr": "Bloc d'introduction avec image"
+			},
+			"status": "stable",
```

Comment:
There is no stability indicator because a design pattern out of scope of our versioning api

```suggestion
```